v4/log/log.go

```diff
@@ -50,6 +50,7 @@ func InitAppLogger(cfg *Config) error {
 		return errors.WithStack(err)
 	}
 	appLogger = Logger{logger}
+	logger = appLogger.WithOptions(zap.AddCallerSkip(1))
```

Comment:
I don't think this has any effect after `appLogger` has been assigned. This line should be placed above line 52.

Reply:
ok